src/redux/modules/queues/actions.js

```diff
@@ -4,12 +4,11 @@ import {
   mergeMap,
   pluck,
   catchError,
-  zip, tap,
 } from 'rxjs/operators';
 import { combineEpics, ofType } from 'redux-observable';
 import { AsyncStorage } from 'react-native';
 import { authGetJSON, errorHandler } from '../../../lib/api';
-import { apiUrl, QUEUE } from '../../../constants/config';
+import { apiUrl } from '../../../constants/config';
```

Comment:
I left this there by accident. Maybe it'd be better to use the constant at line `50`. Or to remove it from `config`